atomics/T1558.003/T1558.003.yaml

```diff
@@ -48,12 +48,12 @@ atomic_tests:
       default: rubeus_output.txt
     rubeus_url:
       description: URL of Rubeus executable
-      type: url
+      type: Url
       default: https://github.com/morgansec/Rubeus/raw/de21c6607e9a07182a2d2eea20bb67a22d3fbf95/Rubeus/bin/Debug/Rubeus45.exe
     flags:
       description: command flags you would like to run (optional and blank by default)
       type: String
-      default: 
+      default: null
```

Comment:
I think this might need to be an empty string instead of null. e.g. ""

Reply:
Hey Carrie, do you know why one should be used over the other?  I'm assuming something to do with `Invoke-AtomicRedTeam` but curious of the use case.

Reply:
Actually, I didn't realize that YAML considers null to actually be null and not just the string "null". so we are good to go on this.